pytensor/link/c/cmodule.py

```diff
@@ -2379,6 +2379,10 @@ def join_options(init_part):
         if sys.platform == "darwin":
             # Use the already-loaded python symbols.
             cxxflags.extend(["-undefined", "dynamic_lookup"])
+            # XCode15 introduced ld_prime linker. At the time of writing, this linker
+            # leads to multiple issues, so we supply a flag to use the older dynamic
+            # linker: ld64
+            cxxflags.append("-ld64")
```

Comment:
Is this back compatible?

Reply:
I’m not sure

Reply:
I’ll have to add a platform version check to add this flag or not